pandas/io/stata.py

```diff
@@ -432,7 +432,8 @@ def parse_dates_safe(dates, delta=False, year=False, days=False):
         d = parse_dates_safe(dates, year=True)
         conv_dates = d.year
     else:
-        raise ValueError("fmt %s not understood" % fmt)
```

Comment:
Revered this to ValueError since it should only raise if the type doesn't exist.